assets/js/translator.js

```diff
@@ -208,6 +219,8 @@ if(modeEnabled('translation')) {
 
                 refreshLangList(true);
                 muteLanguages();
+                refreshChainGraph();
```

Comment:
Seems awfully like you could use a helper function that literally calls these two functions: `refreshChainGraphandChosenPath`? Also, I'm somewhat pulled away from the word 'chosen'. How do you feel about replacing all the "chosen" with "selected" and equivalents?

Reply:
Yup, I'll make this into a helper function. "Chosen" and "selected" are almost equivalent in my opinion; if there's a specific reason you want "selected", we'll go with that.